contrib_bots/lib/githubcomment.py

```diff
@@ -0,0 +1,166 @@
+# See readme.md for instructions on running this code.
+import requests
+import json
+import os
+
+index_error_happened = False
```

Comment:
Instead of a global variable this could be something that you pass to the relevant functions, or a custom exception you raise to avoid further processing the response.